packages/server/lib/browsers/webkit-automation.ts

```diff
@@ -256,15 +299,33 @@ export class WebKitAutomation {
 
     return normalizeGetCookieProps(cookie)
   }
-
-  private async clearCookie (filter: CookieFilter) {
+  /**
+   * Clears one specific cookie
```

Comment:
nice: we don't have docs like this in many places

Reply:
IMO we should do this more often, it's hard to get a read on function expectations without some sort of doc.